bin/extract_family_reps.py

```diff
@@ -48,7 +48,7 @@ def extract_first_sequences(msa_folder, metadata_file, out_fasta):
             "# format: \"csv\"\n"
             "# plot_type: \"table\"\n"
         )
-        csv_writer = csv.writer(csv_out)
+        csv_writer = csv.writer(csv_out, quoting=csv.QUOTE_NONNUMERIC)
```

Comment:
I love csv/tsv files.. but they are poor when you need deal with numbers of dates.